### PR TITLE
added TextTovoiceConverter

### PR DESCRIPTION
**Pull request by Umashankar08**
## Related Issue
#110 
## Email id used to regsiter for VSoc'24
humashankar2003@gmail.com

## Description

Implemented Text to voice converter using Html-Css-Js. A text-to-voice converter is an application or tool that converts written text into spoken audio. It utilizes text-to-speech (TTS) technology to generate human-like speech from input text, enabling users to listen to written content instead of reading it.

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Security enhancement
- [ ] Other (specify): _______________

check in issue by entering [X] in boxes

## Screenshots / Videos (if applicable)

Screenshot 2024-06-20 165155

## Checklist
- [ ] I have performed a self-review of my code.
- [ ] I have read and followed the Contribution Guidelines.
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have followed the code style guidelines of this project.
- [ ] I have checked for any existing open issues that my pull request may address.
- [ ] I have ensured that my changes do not break any existing functionality.
- [ ] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [ ] I have read the resources for guidance listed below.
- [ ] I have followed security best practices in my code changes.

check in issue by entering [X] in boxes
## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Contribution Guidelines

Thank you for considering contributing to our project! To ensure smooth collaboration and effective contribution management, please adhere to the following guidelines:

### Issue Creation

1. **Limit on Issues:**
   - Each contributor is allowed to create a maximum of **4 issues per day**. This helps us manage and address issues efficiently.

### Contribution Levels

2. **Basic Contributions:**
   - This project is primarily focused on documentation. Most of the setup has been completed, so contributors will generally need to work on basic code tasks, such as writing tests.
   - For these tasks, issues will be assigned the **Easy** label.

3. **Acknowledging Hard Work:**
   - If a contributor puts in significant effort on a task, the issue will be upgraded to **Medium**. This is our way of recognizing and appreciating extra effort.

4. **Feature Additions and Component Work:**
   - Contributors working on new features or components using JSX/TSX will be assigned a level based on the complexity and quality of their work.
   - The more complex and valuable the contribution, the higher the level assigned.

### Level Definitions

- **Easy:**
  - Tasks are straightforward, such as fixing minor bugs, writing tests, or making simple documentation updates.
- **Medium:**
  - Tasks require more effort, such as addressing complex bugs, improving existing features, or making substantial documentation improvements.
- **Hard:**
  - Tasks are highly complex and involve significant new feature development, major refactoring, or extensive contributions to the project’s core components.

We look forward to your contributions and appreciate your effort in helping us improve the project!

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -268,6 +268,24 @@
...
+            <a href="Budget Tracker/index.html" class="demo-link">Demo</a>
+        </div>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 47</div>
+            <div class="project-name">Budget Tracker</div>
+            <a href="Budget Tracker/index.html" class="demo-link">Demo</a>
+        </div>
+    </div>
+        <div class="table">
```
Comment (dhairyagothi) [inline public/index.html:284]: change day 
make index html properly
PULL main branch first 
budget tracker is showing 3 times

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by dhairyagothi** [comment]
image
 fix this @Umashankar08

**Comment by Umashankar08** [comment]
i have made the changes .Is it reflecting

**Comment by dhairyagothi** [comment]
image

remove this extra one @Umashankar08

**Comment by Umashankar08** [comment]
i made another  request please go through it

**Pull request closed without merging**